app/views/reviews/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th><%= t "tour_name" %></th>
+      <th><%= t "app.review.title" %></th>
+      <th><%= t "app.review.created_at" %></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% if @reviews.length > 0 %>
+      <%= render partial: "reviews/review", collection: @reviews %>
+    <% end %>
```

Comment:
check trường hợp nếu ko có bản ghi nào thì in dòng text thông báo cho người dùng nữa em